sdk/src/main/java/com/bugsnag/android/Configuration.java

```diff
@@ -336,7 +344,7 @@ public void setNotifyReleaseStages(@Nullable String[] notifyReleaseStages) {
      *
      * @param projectPackages a list of package names
      */
-    public void setProjectPackages(String[] projectPackages) {
+    public void setProjectPackages(@NonNull String[] projectPackages) {
```

Comment:
the getter for this is `@Nullable` but the setter is `@NonNull`. As is `get|setNotifyReleaseStages(…)`. Is this intentional? Can they be set to `null` elsewhere internally, but not via the setter?